cfpb/cfgov-refresh
# Categories

## Additions

- Added specific Blog/Newsroom Categories to Related Post options

## Changes

- Updated article template to render all categories

## Testing

- visit [here](http://localhost:8000/blog/you-have-the-right-to-someone-in-your-financial-corner/) and see that blogs now render all categories icons

- Load Refresh DB
 - visit [here](http://localhost:8000/foia-requests/) and see it pulls a hodge podge of posts
 - edit this [page](http://localhost:8000/admin/pages/640/edit/) and in the sidebar Related posts organism select a specific category or either blogs or newsrooms or both! Then Publish this page
  -  visit [here](http://localhost:8000/foia-requests/) and see it pulls a posts related to your selections

## Review

- @kurtw 
- @richaagarwal 

## Screenshots
before:
![screen shot 2016-03-29 at 5 09 27 pm](https://cloud.githubusercontent.com/assets/254877/14128943/8a63dd68-f5f3-11e5-866d-3de89838eb09.png)

edited:
<img width="861" alt="screen shot 2016-03-29 at 9 16 03 pm" src="https://cloud.githubusercontent.com/assets/254877/14128941/7d60bb68-f5f3-11e5-8120-2dfee4ffb08c.png">

after:
<img width="870" alt="screen shot 2016-03-29 at 9 17 06 pm" src="https://cloud.githubusercontent.com/assets/254877/14128954/9ea0ecd0-f5f3-11e5-8ad3-7a4a1a07804a.png">
